darts/models/forecasting/pl_forecasting_module.py

```diff
@@ -29,10 +30,12 @@ def __init__(
         input_chunk_length: int,
         output_chunk_length: int,
         loss_fn: nn.modules.loss._Loss = nn.MSELoss(),
+        torch_metrics: Optional[List[str]] = None,
```

Comment:
What do you think of accepting instances of `torchmetrics.Metric` here (the parent class from TorchMetrics for all built-in and custom metrics, similar to `loss_fn`) instead of strings? Something similar to `loss_fn`.
```suggestion
# import Metric at beginning
from torchmetrics import Metric
...
        torch_metrics: Optional[List[Metric]] = None,
```
Like this we won't have to setup the metrics, and can check the metrics at model creation:

We might have to group the metrics in a [MetricCollection](https://torchmetrics.readthedocs.io/en/latest/pages/overview.html#metriccollection) for PyTorch Lightning to properly identify them as part of the module.